cost/azure/instances_log_analitics_utilization/azure_instance_log_analytics_utilization.pt

```diff
@@ -1,10 +1,16 @@
-name "Azure Instances Utilization from Log Analytics"
+name "Azure Inefficient Instance Utilization using Log Analytics"
 rs_pt_ver 20180301
 type "policy"
-short_description "Gathers instance utilization data from Azure Log Analytics and tags underutilized instances.  See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/azure/instances_log_analitics_utilization) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
-long_description "Version: 1.1"
+short_description "Gathers instance utilization data from Azure Log Analytics from the last 30 days to identify underutilized instances and provides rightsizing recommendations. Once recommendations are generated, instances can be rightsized in an automated manner or after approval. See the [README](https://github.com/flexera/policy_templates/tree/master/cost/azure/instances_log_analitics_utilization) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
```

Comment:
Use this value for the short_description

Checks for instances that are idle for the last 30 days and terminates them after approval.

Reply:
I am using the below short description that i got from the spreadsheet.
"This checks inefficient instance utilization using provided CPU and Memory thresholds.  Instances matching the criteria can be resized after user approval."

Reply:
Yes, I understand.  All policies with the same policy set should have the same value.  This is new behavior for the catalog with we show policy sets.  Please use the value I provided.  thank you.